release/release_gold.sh

```diff
@@ -513,6 +513,20 @@ function tag_release {
 			return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 		fi
 	done
+
+	if [[ "${_PRODUCT_VERSION}" == 7.4.*-u* ]]
+	then
+		LIFERAY_RELEASE_REPOSITORY_OWNER="brianchandotcom"
```

Comment:
@davisaints, can you please make this variable become an internal parameter of the function `_invoke_github_api` instead?

Then, we can start to send the repository owner as a parameter from now on.